packages/tabs/test/tabs-overflow.test.ts

```diff
@@ -362,3 +440,83 @@ describe('calculateScrollTargetForLeftSide', () => {
         ).to.equal(0);
     });
 });
+
+async function repeatScroll(
+    options: {
+        times: number;
+        elementToUpdate: TabsOverflow;
+        elementToScroll: HTMLElement;
+        distanceToReachInIteration: (iteration: number) => number;
+    },
+    iteration = 1
+): Promise<void> {
+    const {
+        times,
+        elementToUpdate,
+        elementToScroll,
+        distanceToReachInIteration,
+    } = options;
+    if (iteration > times) return;
+
+    const distanceToReach = distanceToReachInIteration(iteration);
+
+    await sendKeys({ press: 'Enter' });
+    await elementUpdated(elementToUpdate);
+    await waitUntil(
```

Comment:
I'm wondering if it would make sense to wrap this in a `try`, `catch` block, so we could log error messages. Otherwise, if the test times out, it might be harder to pinpoint what is breaking.

Reply:
The `scroll to ${distanceToReach}` message has this purpose. If one of these tests fails due to the `waitUntil`, a message similar to the one bellow will appear in the console. I think this is enough to pinpoint the exact point where something breaks.
```bash
❌ TabsOverflow > should scroll up to the last item and back in LTR
      Error: Timeout: scroll to 100
```

Reply:
Nice! I agree that is enough.